elixir-format.el

```diff
@@ -125,54 +125,76 @@ Shamelessly stolen from go-mode (https://github.com/dominikh/go-mode.el)"
               (error "Invalid rcs patch or internal error in elixir-format--apply-rcs-patch"))))))))
   )
 
+(defun elixir-mix-path-set? ()
+  (and (not (= (length elixir-format-mix-path) 0)) (not (= (length elixir-format-elixir-path) 0)))
+  )
+
+(defun display-configuration-missing-error ()
```

Comment:
Please rename this `elixir-format-display-missing-configuration-error`

Reply:
done.